DigitalOffice/Services/ProjectService/Validators/WorkersProjectIdsValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class WorkersProjectIdsValidator : AbstractValidator<WorkersIdsInProjectRequest>
+    {
+        public WorkersProjectIdsValidator()
+        {
```

Comment:
Check request not null here

Reply:
![image](https://user-images.githubusercontent.com/38541098/91647901-73979100-ea69-11ea-9070-7e0006c25bb1.png)